cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,26 @@
+import time
+
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        self.stdout.write("Waiting for database...")
+        db_connect = None
+        attempt = 0
+
+        while not db_connect:
```

Comment:
The loop will continue indefinitely until a connection is successful. Consider adding a maximum number of attempts or a timeout to prevent the script from running indefinitely in case of persistent database issues.